modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/data/handler/BatchEnginePortletDataHandlerUtil.java

```diff
@@ -88,6 +89,8 @@ public static Map<String, Serializable> buildExportParameters(
 
 				return sb.toString();
 			}
+		).putAll(
+			parameters
```

Comment:
I would put this as last

Reply:
I came to say the same to avoid overriding the others.